src/components/calculator.js

```diff
@@ -1,42 +1,37 @@
-import react from 'react';
-import calculate from '../logic/calculate';
+import React, { useState } from 'react';
+import Calculate from '../logic/calculate';
 
-export default class Calculator extends react.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleEvent = this.handleEvent.bind(this);
-  }
+export default function Calculator() {
+  const stateObj = {
+    total: '0',
+    next: null,
+    operation: null,
+  };
 
```

Comment:
[OPTIONAL]
- [ ] Let's always favor the use of ES6 syntax. Please  use the arrow function instead of the `function` keyword. eg.
`const Calculator = () => (

);`